packages/nitro-protocol/test/contracts/MultiAssetHolder/claim.test.ts

```diff
@@ -221,6 +243,18 @@ describe('claim', () => {
 
         // Check that each expectedEvent is contained as a subset of the properies of each *corresponding* event: i.e. the order matters!
         expect(eventsFromTx).toMatchObject(expectedEvents);
+
+        // check payouts
+        await Promise.all(
+          Object.keys(payouts).map(async destination => {
+            const address = convertBytes32ToAddress(destination);
+            expect(
+              (await provider.getBalance(address)).eq(
+                ethBalancesBefore[address].add(payouts[destination])
+              )
+            ).toBe(true);
+          })
+        );
```

Comment:
This is a bit tidier.
```suggestion
        await Promise.all(
          Object.keys(payouts).map(async destination => {
            const address = convertBytes32ToAddress(destination);
            const balanceAfter = await provider.getBalance(address);
            expect(balanceAfter.toString()).toEqual(ethBalancesBefore[address].add(payouts[destination]);
          })
        );
```

Reply:
If you're willing, test feedback is even better if you:
1. construct the expected and actual `ethBalancesAfter` for each address
2. assert that the expected and actual are equal

```suggestion
       const expectedAfter = {};
       const actualAfter = {};
        await Promise.all(
          Object.keys(payouts).map(async destination => {
            const address = convertBytes32ToAddress(destination);
            const actualAfter[address]  = await provider.getBalance(address);
            const expectedAfter[address] = ethBalancesBefore[address].add(payouts[destination]
          })
        );
        expect(actualAfter).toBeExactlyTheSameAs(expectedAfter);
```
Then, on a test failure you get more than `Expected x to be True, but got False`.

Reply:
d53a69f